CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.74.0
+
+- Allow running tmt tests with fmf root outside of git root (#1936)
```

Comment:
```suggestion
- Allow configuring tmt tests with fmf root outside of git root (#1936)
```

Not relevant to the CLI itself, but still nice to have it in the tool soon so the configs are parsable by the CLI tool.